csuntechlab/iSTART
# STR-30 | FE: Create Hamburger Menu

# Description
'Write a meaningful description of your task'
 - Should include logout and extend-ability for future pages
 - Should not try to much beyond bootstrap hamburger
# Testing
'What does the reviewer need to do to test your PR?'
 - Login into iSTART through `steve@csun.edu`
   - Make sure that the navbar isn't appearing
- When you successfullly make it to your welcome page, make sure the navbar appears
- Click on the top two buttons to see that the navbar will (for lack of a better term) de-collapse back to it's original state
- Finally click on the `Logout` option to see that the functionality works
# Installation
'Are there any dependencies the reviewer needs to download?'
NOOOOOOOO! 😢
